src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -117,7 +123,12 @@ public function normalize(mixed $object, string $format = null, array $context =
             if (null === $types) {
                 $types = [$operation->getShortName()];
             }
-            $metadata['@type'] = 1 === \count($types) ? $types[0] : $types;
+
+            if($operation->getRemoveJsonLdIdType()) {
```

Comment:
You could use a ternary here, and store the previous call to the getter in a variable to prevent calling the method two times on the hot path (method calls are costly in PHP).